openedx_learning/apps/authoring/components/management/commands/add_assets_to_component.py

```diff
@@ -70,7 +68,7 @@ def handle(self, *args, **options):
 
         created = datetime.now(tz=timezone.utc)
         keys_to_remove = set()
-        local_keys_to_content = {}
+        local_keys_to_raw_content = {}
```

Comment:
Instead of `local_keys_to_raw_content`, can we use `local_keys_to_content_bytes`?